omod/src/main/resources/config.xml

```diff
@@ -183,6 +183,37 @@
 		<defaultValue>100</defaultValue>
 	</globalProperty>
 
+    <globalProperty>
+        <property>ugandaemr.dsdm.allowClinicalOverrideDSDMPatientStability</property>
+        <description>When set to true, this allows to override the decision made by the system about DSDM Stability </description>
+        <defaultValue>false</defaultValue>
+        <dataType>org.openmrs.customdatatype.datatype.BooleanDatatype</dataType>
+    </globalProperty>
+
+    <globalProperty>
+        <property>ugandaemr.dsdm.currentRegimenDurationRequirementInMonths</property>
+        <description>The number of months when on current regimen required for patient to be categorized as stable</description>
+        <defaultValue>6</defaultValue>
+    </globalProperty>
+
+    <globalProperty>
+        <property>ugandaemr.dsdm.viralloadSuppressionCopies</property>
+        <description>The highest value for a suppression viral load for a patient</description>
+        <defaultValue>1000</defaultValue>
+    </globalProperty>
+
+    <globalProperty>
+        <property>ugandaemr.dsdm.minmunNoOfMonthsOnTBTreatmentRequired</property>
+        <description>The highest value for a suppression viral load for a patient</description>
+        <defaultValue>2</defaultValue>
+    </globalProperty>
+
+    <globalProperty>
```

Comment:
@slubwama This will depend on the Viral Load algorithm, whether, adults (12 months), children, or pregnant (6 months)

Reply:
Those dependencies are not yet clear but as per the guidelines this is not to worry about now